scripts/screeps_rl_live_dashboard.py

```
@@ -300,12 +304,14 @@ def run_refresh_cycle(self, reason: str) -> JsonObject:
                 active_reason=f"{reason} summary",
             )
         if refresh_succeeded(refresh):
+            summary_primed = False
             try:
                 self.prime_summary_cache()
+                summary_primed = True
             except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)